projects/plugins/jetpack/class.json-api.php

```diff
@@ -425,8 +425,8 @@ public function serve( $exit = true ) {
 		// Normalize path and extract API version.
 		$this->path = untrailingslashit( $this->path );
 		preg_match( '#^/rest/v(\d+(\.\d+)*)#', $this->path, $matches );
-		$this->path    = substr( $this->path, strlen( $matches[0] ) );
-		$this->version = $matches[1];
+		$this->path    = substr( $this->path, strlen( $matches[0] ?? '' ) );
+		$this->version = $matches[1] ?? null;
```

Comment:
Seems like it might be clearer to do something like this (which GH won't let me make a suggestion, sigh):
```php
		if ( preg_match( '#^/rest/v(\d+(\.\d+)*)#', $this->path, $matches ) ) {
			$this->path    = substr( $this->path, strlen( $matches[0] ) );
			$this->version = $matches[1];
		}
```

Reply:
Thanks! a5c02c4d68